testrtc.py

```diff
@@ -27,13 +31,49 @@ def get(self):
     content = template.render({})
     self.response.out.write(content)
 
+class UploadHandler(blobstore_handlers.BlobstoreUploadHandler):
```

Comment:
We need to make sure that only visitors of test.webrtc.org are allowed to upload as we do not want anyone to be able to upload reports here (e.g. someone using test.webrtc as their temporary pastebin). Probably done by checking the http referer.